dir.props

```diff
@@ -4,7 +4,12 @@
   <!-- NuGet Information -->
   <PropertyGroup>
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
-    
+
+    <GeneratedPropsDir>$(RepoRoot)/artifacts/obj</GeneratedPropsDir>
```

Comment:
I don't think we need these 3 in dir.props. These properties are very specific to the bootstrapping scenario. Can we move them to a separate props file that is pulled in by dir.props? I don't want to create a "dumping ground of mixed responsibilities".